Makefile

```
@@ -47,8 +47,8 @@ trim-dictionaries:
 check-manifest:
 	check-manifest
 
-check-distutils:
-	python setup.py check --restructuredtext --strict
+check-pyproject:
```

May as well leave some backwards compatibility...
```suggestion
check-distutils: check-pyproject

check-pyproject:
```